Bio/SeqIO/InsdcIO.py

```diff
@@ -745,6 +745,12 @@ def write_record(self, record):
         descr = record.description
         if descr == "<unknown description>":
             descr = "."
+
+        # The DEFINITION filed must end with a period
+        # see ftp://ftp.ncbi.nih.gov/genbank/gbrel.txt [3.4.5]
+        # and discussion https://github.com/biopython/biopython/pull/616
+        # So let's add a period
+        descr += '.'
```

Comment:
If the description was the dummy value ``<unknown description>`` we don't want to record ``..`` but just `.`